.swiftlint.yml

```diff
@@ -42,6 +42,9 @@ variable_name:
     - rs
     - URI
     - URL
+type_name:
+  excluded:
+    - T
```

Comment:
Are you using this somewhere or just updating the file in a more general sense?  I couldn't find the use.